src/ImageLoaders/XexLoader/Structures.cs

```diff
@@ -28,307 +28,340 @@
 
 namespace Reko.ImageLoaders.Xex
 {
-    public class Structures
-    {
-        public const UInt32 XEX2_MAGIC = 0x58455832; //XEX2
+	public class Structures
+	{
+		public const UInt32 XEX1_MAGIC = 0x58455831; //XEX1
```

Comment:
These are tabs. Should not be spaces?

Reply:
i'm struggling with VS2019 not honoring .editorconfig for existing files
i had to use VSCode with an extension to reformat the code...
i don't think it was doing this with VS2017